collect_out_of_range_parameters_DE.py

```diff
@@ -1,9 +1,11 @@
 from report_breach_level_DE_warning import *
 
-def collect_out_of_range_parameters_DE(parameters_exceeded_limit,parameter_name,parameter_value,parameter_min,parameter_max,lang):
+def collect_out_of_range_parameters_DE(parameters_action,parameters_exceeded_limit,parameter_name,parameter_value,parameter_min,parameter_max,lang):

```

Comment:
The `parameters_action` had to be repeated in the DE and EN files. In future, every change (like adding another warning level - early warning and critical warning) would need to be repeated. Adding more languages is also more repetition.
Think of reducing the repetition - Separate the conclusion as data (machine readable, language independent strings) and translate them separately into human readable strings

Reply:
I have removed the duplication by specifying the constants for the language, alert and the collector.

![image](https://user-images.githubusercontent.com/13776900/111633215-44851900-881b-11eb-8550-fb594e3b1b93.png)